Samples/BackgroundBlur/BackgroundBlur/common/mfutils.h

```diff
@@ -0,0 +1,188 @@
+//////////////////////////////////////////////////////////////////////////
+//
+// mfutils.h : Miscellaneous helper functions for Media Foundation.
+// 
+// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
+// PARTICULAR PURPOSE.
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+//////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include <mftransform.h>
+
+namespace MediaFoundationSamples
+{
+    const MFTIME ONE_SECOND = 10000000; // One second in hns
+    const LONG   ONE_MSEC = 1000;       // One msec in hns 
+
+
+    /*
+        MFTimeToMsec
+
+        CreateMediaSource
+        GetCollectionObject
+        GetEventObject
+        GetStreamMajorType
+        GetStreamCurrentMediaType
+        GetTopoNodeObject
+    */
+
+
+    // MFTimeToMsec: Convert 100-ns time to seconds.
+    inline LONG MFTimeToMsec(const LONGLONG& time)
+    {
+        return (LONG)(time / (ONE_SECOND / ONE_MSEC));
+    }
```

Comment:
Comments are misleading (1000ns is 1us instead of 1ms, I think all instances of ns should be us)